spec/models/setting_spec.rb

```diff
@@ -169,4 +169,40 @@
     end
   end
 
+  describe ".clear_cache!" do
+    before do
+      # prime the cache
+      Setting.host_name = 'foo'
+    end
+
+    it "doesn't fetch the setting from the database again" do
+      expect(Setting).to receive(:find_by_name).with('host_name').once
+
+      Setting.host_name
+      Setting.host_name
+    end
+
+    it "fetches the setting again when the cache was cleared" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.